src/Security/Security.php

```diff
@@ -266,15 +254,7 @@ public function createRestrictedUser($id, array $body, array $options = [])
     public function createFirstAdmin(array $body = [], array $options = [])
     {
         if (empty($body)) {
-            throw new InvalidArgumentException('Kuzzle\Security::createFirstAdmin: Unable to create first admin: no id or content specified');
-        }
-
-        if (empty($body['content'])) {
-            throw new InvalidArgumentException('Kuzzle\Security::createFirstAdmin: Unable to create first admin: body["content"] is required');
-        }
-
-        if (empty($body['credentials'])) {
-             throw new InvalidArgumentException('Kuzzle\Security::createFirstAdmin: Unable to create first admin: body["credentials"] is required');
+            throw new InvalidArgumentException('Kuzzle\Security::createFirstAdmin: Unable to create first admin: no id or body specified');
```

Comment:
1. erroneous message, it should be "no body specified"
2. since there is no required body property anymore, then we should be able to provide an empty body